### ISSUE TITLE
why fixing batch norm layer during training?

### ISSUE THREAD

**Issue by xjtuljy** [NVIDIA/retinanet-examples#52]
from retinanet/train.py, the batch norm layer was fixed:
`model = convert_fixedbn_model(model)`

Why is the batch norm layer fixed during training?

**Comment by xjtuljy** [comment]
I just commented out this line to update batchnorm para in training.
